doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+:::
+
+## Global and Local Insights
+
+Insights can be categorised into two types. Local and global. Intuitively, a local insight says something about a single prediction that a model makes. For example, given an image classified as a cat by a model, a local insight might give the set of features (pixels) that need to stay the same for that image to remain classified as a cat. Such an insight provides an idea of what the model is looking for when classifying a specific instance into a particular class.
+
+On the other hand, global insights refer to the behaviour of the model over a set of inputs. Plots showing how a regression prediction varies for a given feature while factoring out all the others are examples. These insights provide a more general understanding of the relationship between inputs and model predictions.
+
+__TODO__:
+- Add image to give idea of Global and local insight
+
+## Insights
+
+Alibi provides several local and global insights with which to explore and understand models. The following gives the practitioner an understanding of which explainers are suitable in which situations.
+
+### Local Necessary Features:
+
+Given a single instance and model prediction, local necessary features are local explanations that tell us what minimal set of features needs to stay the same so that the model still gives the same or close prediction.
+
+In the case of a trained image classification model, local necessary features for a given instance would be a minimal subset of the image that the model uses to make its decision. A machine learning engineer might use this insight to see if the model concentrates on the correct features. Local necessary features are particularly advantageous for checking erroneous model decisions.
+
+The following two explainer methods are available from alibi for generating Local Necessary Features insights. Each approaches the idea in slightly different ways. The main difference is that anchors give a picture of the size of the area over the dataset for which the insight applies, whereas pertinent positives do not.
+
+#### Anchors
+
+| Model-types | Task-types     | Data-types  |
+| ----------- | -------------- | ----------- |
+| Black-box   | Classification | Tabular     |
+|             |                | Text        |
+|             |                | Image       |
+|             |                | Categorical |
+
+Anchors are a local blackbox method introduced in [Anchors: High-Precision Model-Agnostic Explanations](
+https://homes.cs.washington.edu/~marcotcr/aaai18.pdf).
+
+Let A be a rule (set of predicates) acting on such an interpretable representation, such that $A(x)$ returns $1$ if all its feature predicates are true. An example of such a rule, $A$, could be represented by the set $\{not, bad\}$ in which case any sentence, $s$, with both $not$ and $bad$ in it would mean $A(s)=1$.
```

Comment:
A --> $A$

Reply:
The first sentence is not very clear:
 - what is a "set of predicates"?
 - what is "such an interpretable representation"?
 - what are "feature predicates"

Second sentence needs clarifying that the example is text data.